vue/components/ui/templates/error-part/error-part.vue

```diff
@@ -1,12 +1,19 @@
 <template>
     <div class="error-part">
         <button-icon
-            class="button-icon-back"
+            class="button-icon back"
             v-bind:icon="'arrow-left'"
             v-bind:size="60"
             v-if="navigation"
             v-on:click="goBack"
         />
+        <button-icon

```

Comment:
Let's go with allowing the specific logo (passed via prop) instead of using a generic icon.

Another possibility is trying to dynamically require `/res/logo.svg` and fallback to an icon if it doesn't exist (convention over configuration). Summing up:
0. Activate this button through a boolean prop
1. Try dynamic require
2. Use logo passed via prop
3. Fallback to this button icon

![image](https://user-images.githubusercontent.com/3048875/122593005-f15e7480-d05c-11eb-80ed-789d7436474d.png)

Reply:
Currently `<icon>` and `<button-icon>` only support string icon name as a prop.
A dynamic svg icon is possible by including the desired icon in project's own icon assets folder and pointing to it's name in the `icon` prop.
I see the benefit in requiring a dynamic icon as the choices might not satisfy the needs but not the benefit of requiring from paths other than the icon path such as `/res/logo.svg`.

Also, this seems looks far from pulse's header implementation:
<img width="1841" alt="image" src="https://user-images.githubusercontent.com/24736423/122749456-4a105600-d285-11eb-9064-6a07c942daf8.png">
<img width="1841" alt="image" src="https://user-images.githubusercontent.com/24736423/122749565-6613f780-d285-11eb-873c-84d190a0557a.png">



I think for being able to dynamically require any asset from any other folder will require a components-vue refactor as currently it [only accounts for the icon name](https://github.com/ripe-tech/ripe-components-vue/blob/master/vue/components/ui/atoms/icon/icon.vue#L106).

I pushed a change to allow the use of custom icon which satisfies 2nd and 3rd point. The 1st point seems to require a lot of refactoring work for not so much gain. Is this what's intended or did I misunderstood something?